frontend/src/navbar.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navbar.css'
+
+export default function Navbar() {
+  return (
+  <header>
+    <h1 id="heading">Jonathan's Indonesian Specialties</h1>
+    <nav class="topnav">
```

Comment:
Although class is working in this case, we want to use className instead of class in React since class doesn't consistently work in React.